app/assets/stylesheets/timestamp.scss

```diff
@@ -0,0 +1,138 @@
+.timestamp {
+  background: #eeeeee75;
+  padding: govuk-spacing(3) govuk-spacing(2) govuk-spacing(3) govuk-spacing(4);
+  margin-top: 60px;
+  border-left: govuk-spacing(1) solid #347ca9;
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 14px;
+  font-size: .875rem;
+  line-height: 1.1429;
+
+  @include govuk-media-query($from: tablet) {
+    font-size:16px;
+    font-size: 1rem;
+    line-height: 1.25;
+
+    a {
+      font-family: sans-serif;
+    }
+
+    &--inverse-padded dl {
+      margin: 15px;
+    }
+
+    &__term {
```

Comment:
So where are the `__term` and `__definition` suffixes used now, do you still need this